sequel/where.js

```diff
@@ -298,7 +298,7 @@ WhereBuilder.prototype.complex = function complex(queryObject, options) {
 
       queryString += '(SELECT ';
       selectKeys.forEach(function(projection) {
-        var projectionAlias = _.find(_.values(self.schema), {tableName: projection.table}).identity;
+        var projectionAlias = _.find(_.values(self.schema), {tableName: projection.table}).tableName;
```

Comment:
Same deal– this doesn't appear to represent an alias.  Also, isn't this equivalent to `projection.table`?